src/main.js

```diff
@@ -135,6 +135,10 @@ function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 };
 
+function pushToArray(array, asset) {
+  array.push(asset);
```

Comment:
Do you really need a separate function for this?

Reply:
Good point. We were told to break it down if we could but It really only gets used in the randomize poster function.